ts/parsers/elementparser.ts

```diff
@@ -97,6 +97,12 @@ class PowerpointElementParser {
             }
             this.element = rawElement;
             const specialityType = ShapeParser.determineSpecialityType(this.element);
+
+            //throwout unsupported content: for example charts, graphs etc
+            if(specialityType === SpecialityType.None) {
```

Comment:
this is the one flooding the logs then?

Reply:
yes, also is the case for some of the empty shapes with no text.

Reply:
yes saw that, thanks for the comment on that line .. really weird that shapes also come through as paragraphs

Reply:
yes saw that, thanks for the comment on that line .. really weird that shapes also come through as paragraphs

Reply:
Actually the speciality type is something we defined ourselves, and since we do no support shapes, i just look for the text content inside the node, if there is i call it a paragraph, most of the shapes have the text property in it. but some don't. 
Also the similar check was added for those cases, where the shape had the text property so it identified as a paragraph, but there was no content inside those paragraphs, so i discarded those elements all together.